module.xml

```diff
@@ -24,5 +24,23 @@
   <Invoke Class="%IPM.Utils.Migration" Method="RunAll">
     <Arg>${verbose}</Arg>
   </Invoke>
+
+  <!-- List of python wheels for offline installation – oras and its transitive dependencies -->
+  <PythonWheel Name="attrs-25.1.0-py3-none-any.whl" ExtraPipFlags="--no-deps"/>
```

Comment:
We need to double check that this doesn't break with zpm "install 0.10.0" while running 0.9.x - I think we'll be OK though, and CI should have caught this already if it'd be an issue.

Reply:
@isc-tleavitt  Do you have a suggestion for the best way to test this?

Reply:
Spin up a fresh container, install 0.9.2, upgrade to 0.10.0, see if ORAS works.

Reply:
(I might have time for this today.)